fxgl-input/src/main/kotlin/com/almasb/fxgl/input/virtual/VirtualInput.kt

```diff
@@ -18,32 +18,41 @@ import javafx.scene.shape.*
 import javafx.scene.text.Font
 import javafx.scene.text.Text
 
+private const val ARROW_CIRCLE_SIZE = 25.0*2;
+
+private const val XBOX_OUTER_CIRCLE_SIZE = 25.0*2;
+private const val XBOX_INNER_CIRCLE_SIZE = 18.0*2;
+
+private const val PS_INNER_CIRCLE_SIZE = 25.0*2;
```

Comment:
The order is different: `XBOX_OUTER_CIRCLE_SIZE ` vs `PS_INNER_CIRCLE_SIZE `. The outer one should be larger. Please check. Suggested fix: rename `PS_INNER_CIRCLE_SIZE ` to outer and outer to inner ...